Added YouTube video to index.md on Time Series (#85188)
grafana/grafana: docs/sources/panels-visualizations/visualizations/time-series/index.md

You can migrate from the old Graph visualization to the new time series visualization. To migrate, open the panel and click the **Migrate** button in the side pane.
{{% /admonition %}}

> The following video provides a visual guide through the creation steps and common customizations of Time Series panels. The best help for beginners creating their first Time Series panels.

Reviewer: ```suggestion
The following video guides you through the creation steps and common customizations of time series visualizations and is great for beginners:
```

Author: Excellent! Thank you so much!
